eo-maven-plugin/src/main/java/org/eolang/maven/OptimizeMojo.java

```diff
@@ -141,12 +153,24 @@ public void exec() throws IOException {
      * Optimize XML file after parsing.
      *
      * @param file EO file
+     * @param tojo Source tojo
      * @return The file with optimized XMIR
-     * @throws FileNotFoundException If fails
+     * @throws IOException If fails
      */
-    private XML optimize(final Path file) throws FileNotFoundException {
+    @SuppressWarnings({"PMD.AvoidCatchingGenericException", "PMD.ExceptionAsFlowControl"})
+    private XML optimize(final Path file, final Tojo tojo) throws IOException {
```

Comment:
@potatmen here we can pass only `tojo` and extract `file` within the method. Currently the parameter are excessive.